File: 프로그래머스/0/181862. 세 개의 구분자/세 개의 구분자.js
function solution(myStr) {
    const result = myStr.replaceAll('a', ' ')
        .replaceAll('b', ' ')
        .replaceAll('c', ' ')
        .split(' ')
        .filter(str => str !== '');

    return result.length > 0 ? result : ["EMPTY"];
}

File: 프로그래머스/0/181862. 세 개의 구분자/세 개의 구분자.java
import java.util.Arrays;

class Solution {
    public String[] solution(String myStr) {
        String[] arr = myStr.replace("a", "c")
            .replace("b", "c")
            .split("c");
        
        String[] result = Arrays.stream(arr)
            .filter(s -> !s.isEmpty())
            .toArray(String[]::new);
        
        return result.length > 0 ? result : new String[]{"EMPTY"};
    }
}